.csharpierignore

```diff
@@ -1 +1,2 @@
-Uploads/
\ No newline at end of file
+Uploads/
+**/node_modules/*.cs
```

Comment:
This is actually the ignore file for csharpier itself, the code change would probably end up going in https://github.com/belav/csharpier/blob/master/Src/CSharpier.Cli/CommandLineFormatter.cs